src/imp/tracer_imp.js

```diff
@@ -883,11 +883,13 @@ export default class TracerImp extends EventEmitter {
         let clockOffsetMicros = this._clockState.offsetMicros();
 
         // Diagnostic information on the clock correction
-        this.logStable('cr/time_correction_state', {
-            offset_micros  : clockOffsetMicros,
-            active_samples : this._clockState.activeSampleCount(),
-            ready          : clockReady,
-        });
+        if (this._options.debug) {
```

Comment:
> As far as I understand, mocha is not supposed to run tests in parallel.

I had the impression that mocha runs tests in parallel because this exists:

https://www.npmjs.com/package/serial-mocha

... but, then again, this also exists:

https://www.npmjs.com/package/mocha-parallel-tests

You also said:

> In other words, there will be two separate instances of Node code running (the host and the child process) -- but they are part of one test. Perhaps that's what you were seeing?

Would it be surprising for the two separate node instances/processes to share the same `Tracer` singleton? (Not sure how literal "process" is here)

TL;DR, though, I think the easiest thing would be to spend like 15 minutes tomorrow morning pairing on this and I can show you the various odd things I was seeing. Gracias...